renv.lock

```diff
@@ -146,19 +146,19 @@
     },
     "assessr": {
       "Package": "assessr",
-      "Version": "0.5.2",
+      "Version": "0.6.0",
```

Comment:
`assessr` got updated as a result of updating `ccao` to the most recent version.